.github/workflows/publish-pypi.yml

```diff
@@ -0,0 +1,20 @@
+name: publish-PyPI CD
+
+on:
+  workflow_run:
+    workflows: ["metadata-extractor CI"]
+    branches: [master]
+    types: 
+      - completed
+
+jobs:
+  release:
+    steps:
+    - name: Publish package to PyPI
+      env:
+        VERSION: ${{ github.event.release.tag_name }}
```

Comment:
This might not be sufficient yet, since we aren't using GitHub tags (yet) and so what will happen is that this action will run for every commit and try to publish with the last `tag_name` and fail.